src/features/packageManager/components/PackageManager.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo, useState } from "react";
```

Comment:
@ostojics @pierrotsmnrd I think there is some confusion here around the name/concept of `PackageManager`. Looking at the figma, I'm pretty sure `Package Manager` is just placeholder text for the name of our App. In other words, all references to `PackageManager` here should just be something like `EnvironmentSearch` instead, and the `packageManager` dir should just be `environments` instead

Reply:
I see, that makes sense